gax-grpc/src/test/java/com/google/api/gax/grpc/ProtoOperationTransformersTest.java

```diff
@@ -29,27 +29,26 @@
  */
 package com.google.api.gax.grpc;
 
+import static com.google.common.truth.Truth.assertThat;
```

Comment:
Don't static import this since there are at least three different assertThat's this could be

Reply:
Isn't this the recommended way to use Truth?
https://truth.dev/#2-add-static-imports-for-truths-entry-points

Reply:
I actively disagree with that recommendation. I like to know where my methods come from. In this case I was about to send you a completely wrong review because I assumed it was a different `assertThat()` method than it actually was. As I said, there are at least three in common use. Removing ambiguity is important for readability.

Reply:
@elharo Thanks for the review!!

I have removed static `Truth.assertThat` import introduced in this PR, Please have another look.